packages/internal/src/callback.js

```diff
@@ -14,8 +14,8 @@ const ownKeys =
   );
 
 /**
- * @template T
- * @typedef {(...args: Parameters<ReturnType<prepareAttenuator>>) => Farable<T>} MakeAttenuator
+ * @template {import('@endo/exo/src/exo-makers.js').Methods} T
```

Comment:
What would it take to make the following work?
```suggestion
 * @template {import('@endo/exo').Methods} T
```

Reply:
A release of Endo that exported the types at the root. I assume Endo maintainers haven't done this yet because they haven't decided on a stable set of type exports.

<img width="841" alt="Screenshot 2023-12-23 at 9 19 09 AM" src="https://github.com/Agoric/agoric-sdk/assets/21505/24a9594a-115c-4ea7-a989-aec33dd5a3ef">